packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.tsx

```diff
@@ -382,15 +399,17 @@ export const ManageVersions = (props: ManageVersionsProps) => {
             </Tabs.TabList>
 
             <Tabs.Panel value={ManageVersionsTabs.Versions.toString()}>
-              <VersionsTab
-                status={versionStatus}
-                onVersionUpdated={refreshVersions}
-                loadMoreVersions={getMoreVersions}
-                onViewClick={onViewClick}
-                tableData={versionsTableData ?? []}
-                changesetClient={changesetClient}
-                setRelatedChangesets={setRelatedChangesets}
-              />
+              {_currentTab === ManageVersionsTabs.Versions && (
```

Comment:
why this is needed?

Reply:
doing this to fix this issue: 
![image](https://github.com/iTwin/admin-components-react/assets/148448690/b798f6f7-9b33-46f9-82f1-9e27635512e7)

Reply:
In the Previous v2 version HorizontalTab was used and conditional rendering was done based on _currentTab, but now using Tab.Panel and it doesn't rerender the component on tab change and as a result table data is not reset.

Checked on some other portals , we don't retain changes on previous tab content on tab changes.
Any suggestion to address this issue?

Reply:
@mayank99 Probably should document, that with new tabs api tabs will be hidden rather than unmounted (which was the case before).

Reply:
Could you try adding `key={hidden}` to the old `Tabs.Panel` and see if it fixes the re-rendering issue? If it does, then we can make this change within iTwinUI and also document it.

We would like to avoid the need for users to reach for legacy API (which is only kept for backwards compatibility purposes).

Reply:
On adding key={_currentTab} to Tab.Panel fixes the issue.